src/serialization-versions.ts

```diff
@@ -0,0 +1,4 @@
+export enum SerializationFormat {
+    legacy = 'legacy',
+    latest_version = 'latest_version'
```

Comment:
Is this taken from the Ruby/Elixir version? 

Because if not I think maybe we be explicit and call this `'bson'`

Reply:
Yes, this is in line with the Ruby and Elixir ports

Reply:
Ok, happy to leave then